### PR TITLE
Have circle.yml now call the makefile to install...

### PR DESCRIPTION
**Pull request by rargulati**
...this dependency. The makefile downloads a script from a repo
that we own (shyp/set-node-npm). This pattern emerged from work in shyp_api.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,9 @@
 MOCHA_OPTS= --check-leaks
 REPORTER = dot
 
-test: test-unit test-integration
```
Comment (kevinburkeshyp) [inline Makefile:6]: Ah- when they're up here, they're calling the other targets as dependencies of this one, essentially, "run the test-unit target and the test-integration target before calling this one"

**Comment by kevinburkeshyp** [comment]
Probably need to clear cache since `pg-native` is a native module.

**Comment by rargulati** [comment]
Good call on `pg-native`. Seems to have done the trick.

**Comment by kevinburkeshyp** [comment]
LGTM

**Pull request merged**